jaseci_ai_kit/setup.py

```diff
@@ -20,7 +20,7 @@ def get_ver():
         "torch>=1.10.2,<2.0.0",
         "pandas>=1.4.1,<2.0.0",
         "flair==0.10",
-        "numpy>=1.22.1,<2.0.0",
+        "numpy==1.23.5",
```

Comment:
Should we pin these so specifically @ypkang , Increases the likeliness of package conflicts, i like something like >=1.23.0,<1.24 a little better

Reply:
yea we can change it to that. I just needed to upgrade numpy because there is a problem with 1.22.* that was causing the docker build to fail from time to time.

Reply:
updated.